### PR TITLE
Add a method to prepare multipath config file

### PR DESCRIPTION
**Pull request by lento-sun**
When device-mapper-multipath installed, it doesn't generate default
config file in /etc/multipath.conf, this will cause some cases failed
due to multipath commands failed. Add a method to prepare the config
file.

Signed-off-by: Yi Sun <yisun@redhat.com>

### CODE REVIEW

File: virttest/utils_npiv.py [hunk 1]
```diff
@@ -341,3 +341,38 @@ def restart_multipathd(mpath_dev="", expect_exist=False):
...
+    :param conf_content: Content of the conf file.
+    :param replace_existing: True means to replace exsiting conf file.
+    :param restart_multipathd: True means to restart multipathd.
+    :return: The content of original conf, can be used to recover env.
+    """
+    default_conf_content = ("defaults {\n\tuser_friendly_names yes"
+                            "\n\tfind_multipaths yes\n}")
+    old_conf_content = ""
+    if not conf_content:
+        conf_content = default_conf_content
```
Comment (chunfuwen) [inline virttest/utils_npiv.py]: Better not change the variable value, which passed in as parameter

File: virttest/utils_npiv.py [hunk 2]
```diff
@@ -341,3 +341,38 @@ def restart_multipathd(mpath_dev="", expect_exist=False):
...
+    if os.path.exists(conf_path):
+        with open(conf_path, 'r') as conf_file:
+            old_conf_content = conf_file.read()
+            logging.info("Old multipath conf is: %s" % old_conf_content)
+        if replace_existing:
+            with open(conf_path, 'w') as conf_file:
```
Comment (chunfuwen) [inline virttest/utils_npiv.py]: I assume this open can be merged into open in line 363

Comment (lento-sun) [reply virttest/utils_npiv.py]: yes, merged with open file with r+ access right, thx

File: virttest/utils_npiv.py (hunk 1 above)
Comment (lento-sun) [reply virttest/utils_npiv.py]: ok, changed to `new_conf_content = conf_content if conf_content else default_conf_content`
thx

File: virttest/utils_npiv.py [hunk 3]
```diff
@@ -341,3 +341,38 @@ def restart_multipathd(mpath_dev="", expect_exist=False):
         if expect_exist:
             return True
     return False
+
+
+def prepare_multipath_conf(conf_path="/etc/multipath.conf", conf_content="",
+                           replace_existing=False, restart_multipath=True):
+    """
+    Prepae multipath conf file.
```
[the remark this answers is not in the document]
Comment (lento-sun) [reply virttest/utils_npiv.py]: thx, updated.

**Pull request merged**